### DISCUSSION TITLE
Compute embeddings best practices?

### DISCUSSION THREAD

**Discussion by Naxdy** [coqui-ai/TTS#1646] [General Q&A] [answered]
Just a quick question regarding the `bin/compute_embeddings.py`

I am trying to replicate training the YourTTS model, only on a higher audio sample rate. I know that the `speakers.json` from the official files is generated using the `compute_embeddings.py`, however it seems to only contain a fraction of the speakers present in the datasets. When I use `compute_embeddings.py` to generate d_vectors, it creates a pretty large file (as is to be expected).

Now I'm just wondering about the best practices here; whether I should keep the huge file that includes embeddings from every audio file, or whether I should only compute a small subset of embeddings?

**Comment by erogol** [comment, marked as answer]
You should compute all of them
